src/sidebar/pin/tree/PinTreeDataProvider.ts

```diff
@@ -49,6 +51,22 @@ class PinTreeDataProvider extends TreeDataProvider {
                 throw new Error("Wrong item type to show pin of");
         }
     }
+
+    public revealScreen(screen: Screen) {
+        // Find screen tree item
+        const screenTreeItem =
+            this.getRoots().find(item => item instanceof ScreenTreeItem && item.screen._id === screen._id);
+
+        this.reveal(screenTreeItem, `Screen: ${screen.name}|${screen._id}`);
+    }
+
+    public revealComponent(component: ResponseZeplinComponent) {
+        // Find component tree item
+        const componentTreeItem = this.getRoots()
+            .find(item => item instanceof ZeplinComponentTreeItem && item.zeplinComponent._id === component._id);
+
+        this.reveal(componentTreeItem, `Component: ${component.name}|${component._id}`);
```

Comment:
shouldn't this be
```suggestion
        this.reveal(componentTreeItem, `Component: ${component.name || component._id}`);
```
or ts excepts yours as well?

Reply:
this is not to log either of them. It is to log both of them in the format of `name|id`

Reply:
A-ha gotcha!